src/combine.cpp

```diff
@@ -2334,6 +2334,12 @@ static void fix_casts(chunk_t *start)
    }
    paren_close = pc;
 
+   if (last == NULL)
+   {
+      /* \todo can we be sure that last is never NULL here? */
```

Comment:
If we can be sure it is never NULL an assert is fine.  But I am not sure about it. Thus my question.